srv/isc-agent/settings.py

```diff
@@ -57,6 +57,10 @@
 )
 DATABASE_SESSION = Session(DATABASE_ENGINE)
 
+# CSS certification key and certificate
+PRIVATE_KEY = os.path.join('key.pem')
+CERT_KEY = os.path.join('cert.pem')
+
 # PLUGINS
 # Eventually this value will be inferred from a settings file of some sort
 PLUGINS = [
```

Comment:
Somewhere in the configs it needs to contain what ports should be used for HTTPS and what ports should be used for HTTP